packages/sequelize-transparent-cache/src/cache/__test__/util.test.js

```diff
@@ -0,0 +1,136 @@
+const sequelize = require('./sequelize')
+
+const { User, Article, Group } = sequelize.models
+
+beforeAll(() => sequelize.sync())
+afterEach(() => (sequelize.sync({ force: true })))
```

Comment:
This is not really UNIT tests, but I think they are good enough to cover this new code. How about just merge this new test cases with existing test suite in `class.test.js`?

Reply:
Should I merge it with the existing test suit in ```class.test.js``` or include it as a different test suit?

Reply:
How about putting it into new `describe` inside existing class.test.js ?

Reply:
Yeah right